Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -241,11 +243,14 @@ public enum RenderBlockContent: Equatable {
         public var metadata: RenderContentMetadata?
 
         /// Creates a new table with the given data.
-        public init(header: HeaderType, rows: [TableRow], extendedData: Set<TableCellExtendedData>, metadata: RenderContentMetadata? = nil) {
+        public init(header: HeaderType, alignments: [ColumnAlignment]?, rows: [TableRow], extendedData: Set<TableCellExtendedData>, metadata: RenderContentMetadata? = nil) {
             self.header = header
             self.rows = rows
             self.extendedData = extendedData
             self.metadata = metadata
+            if let alignments = alignments, !alignments.allSatisfy({ $0 == .unset }) {
```

Comment:
It seems odd to not assign to the property if they're all `unset` here. Is there a technical reason that's needed? As a user of the API, I'd expect my argument to get assigned to the property.

Reply:
This is mainly for backwards compatibility with existing DocC-Render - the absence of the `alignments` list is the same as when they're all `unset`, so i decided to drop it to save on encoding space.

Reply:
Not emitting it in the encoder seems to be handled in the encode function though, right? It doesn't seem like the extra check here is required.

Reply:
I see, then it seems like we should implement this filtering in the decoder, rather than the initializer, that way clients that use the Swift model directly aren't affected by this.